services/slam/builtin/builtin_test.go

```diff
@@ -915,7 +916,7 @@ func TestGetMapAndPosition(t *testing.T) {
 		Sensors:          []string{"good_color_camera"},
 		ConfigParams:     map[string]string{"mode": "mono", "test_param": "viam"},
 		DataDirectory:    name,
-		MapRateSec:       200,
+		MapRateSec:       &validMapRate,
```

Comment:
[q] Did we have to change this because it is now a pointer? It won't accept a number directly? Can you move the `validMapRate` int to the other ints in the `const( )` brackets?

Reply:
Saw now that you and John already talked about this. Ignore if there's no new information regarding this :)